packages/ra-ui-materialui/src/AdminUI.tsx

```diff
@@ -18,20 +19,40 @@ export const AdminUI = ({
     loginPage = Login,
     authCallbackPage = AuthCallback,
     notification = Notification,
+    error: errorComponent,
     ...props
-}: AdminUIProps) => (
-    <ScopedCssBaseline enableColorScheme>
-        <CoreAdminUI
-            layout={layout}
-            catchAll={catchAll}
-            loading={loading}
-            loginPage={loginPage}
-            authCallbackPage={authCallbackPage}
-            {...props}
-        />
-        {createElement(notification)}
-    </ScopedCssBaseline>
-);
+}: AdminUIProps) => {
+    const [errorInfo, setErrorInfo] = useState<ErrorInfo | undefined>(
+        undefined
+    );
+
+    const handleError = (error: Error, info: ErrorInfo) => {
+        setErrorInfo(info);
+    };
+
+    return (
+        <ScopedCssBaseline enableColorScheme>
+            <CoreAdminUI
+                layout={layout}
+                catchAll={catchAll}
+                loading={loading}
+                loginPage={loginPage}
+                authCallbackPage={authCallbackPage}
+                error={({ error, resetErrorBoundary }) => (
```

Comment:
no, pass the `error` prop as is

Reply:
@fzaninotto  Like this?
```suggestion
                error={Error}
```
Not sure to understand (cf [Layout L46](https://github.com/marmelab/react-admin/blob/next/packages/ra-ui-materialui/src/layout/Layout.tsx#L46)). If I do it, it doesn't work anymore.